content/en/docs/tutorials/security/ns-level-pss.md

```diff
@@ -0,0 +1,131 @@
+
+To further debug and diagnose cluster problems, use 'kubectl cluster-info dump'.
+```
+
+# Create Namespace
+
+Create a new namespace `example` for this tutorial:
+
+```
+$ kubectl create ns example
+namespace/example created
+```
+
+# Applying one Pod Security Standard
+
+Enable Pod Security Standards on this namespace using labels supported by
+built-in Pod Security Admission. In this step we will warn on baseline pod
+security standard as per the latest version (default value)
+
+```
+kubectl label --overwrite ns example \
+  pod-security.kubernetes.io/warn=baseline \
+  pod-security.kubernetes.io/warn-version=latest
+```
+
+# Applying multiple Pod Security Standards
+
+Multiple pod security standards can be enabled on any namespace, using labels.
+Following command will `enforce` the `baseline` Pod Security Standard, but
+`warn` and `audit` for `restricted` Pod Security Standards as per the latest
+version (default value)
+
+```
+kubectl label --overwrite ns example \
+  pod-security.kubernetes.io/enforce=baseline \
+  pod-security.kubernetes.io/enforce-version=latest \
+  pod-security.kubernetes.io/warn=restricted \
+  pod-security.kubernetes.io/warn-version=latest \
+  pod-security.kubernetes.io/audit=restricted \
+  pod-security.kubernetes.io/audit-version=latest
+```
+
+# Create Pod
+
+Create a minimal pod in `example` namespace:
+
+```
+$ cat <<EOF > /tmp/pss/nginx-pod.yaml
+apiVersion: v1
+kind: Pod
+metadata:
+  name: nginx
+spec:
+  containers:
+    - image: nginx
+      name: nginx
+      ports:
+        - containerPort: 80
+EOF
+
+$ kubectl apply -n example -f /tmp/pss/nginx-pod.yaml
+Warning: would violate PodSecurity "restricted:latest": allowPrivilegeEscalation != false (container "nginx" must set securityContext.allowPrivilegeEscalation=false), unrestricted capabilities (container "nginx" must set securityContext.capabilities.drop=["ALL"]), runAsNonRoot != true (pod or container "nginx" must set securityContext.runAsNonRoot=true), seccompProfile (pod or container "nginx" must set securityContext.seccompProfile.type to "RuntimeDefault" or "Localhost")
+pod/nginx created
+```
+
+# Clean up
+
+Run `kind delete cluster -name psa-ns-level` to delete the cluster created.
+
+# Bringing it all together
+
+Running the commands in this
+[gist](https://gist.github.com/PushkarJ/c694bac35c2d100f906861667474afb5)
+which were also discussed in this tutorial will do these steps in one go:
+
+1. Create KinD cluster
+2. Create new namespace
+3. Apply `baseline` Pod Security Standard in `enforce` mode while applying
+   `restricted` Pod Security Standard also in `warn` and `audit` mode.
+4. Create a new pod with the following pod security standards applied
+
+> Shoutout to Lachlan Evenson who wrote a
+> [blog](https://medium.com/@LachlanEvenson/hands-on-with-kubernetes-pod-security-admission-b6cac495cd11)
+> that covered similar content with older settings when pod security admission
+> was in alpha state (v1.22)
```

Comment:
@lachie83 let me know if you would rather keep this or remove it in favor of the call out for official v1.23 blog post? cc @jimangel

Reply:
I like the idea of both sources pointing at each other. If we're planning on this landing on `main` post-release, we might consider throwing a `hold` label on this so it doesn't get merged early.

Reply:
@jimangel Yes this is on `/hold` already based on @sftim's suggestion here: https://github.com/kubernetes/website/pull/30502#issuecomment-973931518

The blog links are added here: https://github.com/kubernetes/website/pull/30422#discussion_r752808164 and tutorial links are requested here: https://github.com/kubernetes/website/pull/30502#discussion_r752667699

I can remove the medium blog link if the above cross linking works.